scripts/fit_pgd.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
+from absl import app
+from absl import flags
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import pickle
+import flowpm
+from flowpm import tfpm
+from flowpm.utils import cic_paint
+from flowpm.tfpower import linear_matter_power
+from functools import partial
+import flowpm.scipy.interpolate as interpolate
+
+from nbodykit.cosmology import Cosmology
+from nbodykit.cosmology.power.halofit import HalofitPower
+from astropy.cosmology import Planck15
+import astropy.units as u
+
+flags.DEFINE_string("filename", "results_fit_PGD.pkl", "Output filename")
+flags.DEFINE_integer("niter", 51, "Number of iterations of the first PGD fit")
+flags.DEFINE_integer("niter_refine", 11,
```

Comment:
Seems a little severe to me and might not work out everytime but I guess we are leaving it to be fit and then gauge the results from the plot?

Reply:
yeah... you mean the 11 iterations right? In my test, the parameters change very little from one snapshot to the next, but maybe it's because we have 40 steps